src/lib.rs

```diff
@@ -508,20 +508,6 @@ pub use month::{Month, ParseMonthError};
 mod traits;
 pub use traits::{Datelike, Timelike};
 
-/// Constants that can be used by all components.
-pub mod consts {
-    /// Constants of type `f32`.
-    pub mod f32 {
-        /// Number of weeks in a year.
-        pub const WEEKS_PER_YEAR: f32 = 52.1775;
-    }
-    /// Constants of type `f64`.
-    pub mod f64 {
-        /// Number of weeks in a year.
-        pub const WEEKS_PER_YEAR: f64 = 52.1775;
-    }
-}
```

Comment:
This is not "imprecise". This actually very precise: https://en.wikipedia.org/wiki/Week#Definition_and_duration

Reply:
Well, if you define a `YEAR` to be a "mean year" computed across 400 years, then yes. For the conventional meaning of a year (of either 365 or 366 days) it isn't very precise.

Reply:
So I guess it's just the naming that is incorrect

Reply:
All other things being equal I'd prefer not to expose relatively niche API surface.

Reply:
It's not like the number of weeks in a mean year is going to change though